stories/assets/js/navigation.js

```diff
@@ -0,0 +1,150 @@
+let lang = 'english';
+let subMenus = [];
+export const init = (locale) => {
+  const menuItem = $('.menu-item');
+  const megaNavOption = $('.mega-nav-option');
+  const mobLinkWrapper = $('.mob-links');
+  const submenuli = $('.submenu li');
+  const midNav = $('.mid-nav');
+
+  const getSubMenuData = (id) => {
+    const data = subMenus.find((sub) => sub.link.id === id);
+    if (!data.image) {
+      $('.mega-nav-option .mega-sub-description').removeClass('large-6');
+      $('.mega-nav-option .mega-sub-description').addClass('large-10');
```

Comment:
@Ashish-srij can we write like this $('.mega-nav-option .mega-sub-description').removeClass('large-6').addClass('large-10');